### DISCUSSION TITLE
How to interact with app if no elements exist?

### DISCUSSION THREAD

**Discussion by LouisDeconinck** [appium/appium#16502] [Q&A]
I am trying to scrape some data from a mobile game. When looking at the App Source tree structure in the Inspector it seems empty. How can I still interact with this app (find text, click buttons, ...)?
![Screenshot COS](https://discuss.appium.io/uploads/default/optimized/2X/e/e870303591b371d8a42719123ae77f2a49aab4fa_2_1035x550.png)

Link to app: [https://play.google.com/store/apps/details?id=com.gamingforgood.clashofstreamer](https://play.google.com/store/apps/details?id=com.gamingforgood.clashofstreamer)

**Comment by KazuCocoa** [comment]
I would do with coordinate based interactions with W3C actions http://appium.io/docs/en/commands/interactions/actions/

https://discuss.appium.io/ may also help to ask such questions

**Comment by jlipps** [comment]
There is also http://github.com/projectxyzio/appium-unity-plugin if this is a Unity game. Or use https://github.com/appium/appium-plugins/packages/images to do image-based automation. But yeah in general, games don't have standard UI elements and therefore most test automation frameworks cannot work with them.
